### ISSUE TITLE
🔓 Missing HTTP security headers

### ISSUE THREAD

**Issue by R3gi** [storj/storj#4594]
I noticed the storj.io domain does not use certain http security headers, such as Strict-Transport-Security etc (https://securityheaders.com/?q=storj.io&followRedirects=on). Since Storj claims to be secure I wonder why it does not use such security headers which could prevent potential XSS and other vulnerabilities?

Also the domain isn't HSTS preloaded (for obvious reasons) → https://hstspreload.org/?domain=storj.io

**Comment by stefanbenten** [comment]
Hello @R3gi ,

I relayed this internally and we will get back to it/you as soon as time allows.
It also is attached to the respective Project for track keeping.

**Comment by mobyvb** [comment]
yeah, we have been trying to set up that redirect @R3gi, but haven't yet found a solution that does the job. I think it would be best for us to close this issue and create a separate one regarding the hsts redirect

**Comment by mobyvb** [comment]
new task created https://github.com/storj/storj/issues/4718
